pkg/restapi/vc/operation/operations_test.go

```diff
@@ -1686,7 +1687,8 @@ func TestIssueCredential(t *testing.T) {
 		proof, ok := signedVCResp["proof"].(map[string]interface{})
 		require.True(t, ok)
 		require.Equal(t, cslstatus.Context, signedVCResp["@context"].([]interface{})[1])
-		require.Equal(t, "Ed25519Signature2018", proof["type"])
```

Comment:
should we also have tests for `Ed25519Signature2018`?

Reply:
I will add

Reply:
done